test/blackbox-tests/test-cases/vlib-default-impl/run.t

```diff
@@ -33,29 +33,15 @@ Check that ambiguity is handled correctly.
 Check that default implementation data is installed in the dune package file.
   $ dune build --root dune-package
   Entering directory 'dune-package'
+  File "dune", line 4, characters 25-34:
+  4 |  (default_implementation a-default))
+                               ^^^^^^^^^
+  Error: Library "a-default" not found.
+  Hint: try: dune external-lib-deps --missing --root dune-package @@default
+  [1]
```

Comment:
This test should be updated, otherwise it's not longer testing what it was meant to test.

Reply:
Done.